Lib/test/test_pdb.py

```diff
@@ -3039,7 +3039,7 @@ def test_pdb_issue_gh_108976():
     """
 
 def test_pdb_issue_gh_127321():
```

Comment:
This test is not testing what is supposed anymore. It was carefully constructed to catch a corner case which does not exist anymore. The current behavior is just normal. I think leaving it here is simply misleading. I suggest removing it for good.

Reply:
It's testing a corner case that could show up again as a regression. I once found a bug in version 3.8 because of a test for a corner case that was added when the same but was fixed in 2.6.

Reply:
Sure, then we can leave it here.